package.json

```diff
@@ -18,7 +18,11 @@
     "typescript"
   ],
   "main": "index.js",
-  "browser": "dist/protobuf.js",
+  "browser": {
+    "protobufjs": "./index.js",
```

Comment:
Are these aliases actually necessary? From my understanding, as the library is imported from the outside already, having them on the inside shouldn't do anything? For example, there is a `light.js` at the root of the package already that is picked up on `require("protobufjs/light")`.

Reply:
Removing the browser entry works, and specifying all three works. Specifying only index.js does not work.